### PR TITLE
Ports Trait Rebalance From Floof

### PR DESCRIPTION
**Pull request by ShatteredSwords**
# Description

This PR ports my trait balance changes from Floof. You can find the PR for that repo at https://github.com/Fansana/floofstation1/pull/170.

---

# Changelog

:cl:
- tweak: The cost of many traits have been updated
- tweak: A few traits have been tweaked

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]

**Review by VMSolidus — APPROVED** [review]

File: Resources/Prototypes/Traits/physical.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 - type: trait
   id: WillToDie
   category: Physical
-  points: 2
+  points: 1
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Traits/physical.yml:26]:
```suggestion
  points: 2
```

File: Resources/Prototypes/Traits/physical.yml [hunk 2]
```diff
@@ -40,12 +40,12 @@
         - WillToLive
   components:
     - type: DeadModifier
-      deadThresholdModifier: -15
+      deadThresholdModifier: -10
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Traits/physical.yml:43]:
```suggestion
      deadThresholdModifier: -15
```

File: Resources/Prototypes/Traits/physical.yml [hunk 3]
```diff
@@ -84,7 +84,7 @@
         - Tenacity
   components:
     - type: CritModifier
-      critThresholdModifier: -10
+      critThresholdModifier: -7
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Traits/physical.yml:87]:
```suggestion
      critThresholdModifier: -10
```

File: Resources/Prototypes/Traits/physical.yml [hunk 4]
```diff
@@ -40,12 +40,12 @@
...
   components:
     - type: DeadModifier
-      deadThresholdModifier: -15
+      deadThresholdModifier: -10
 
 - type: trait
   id: Tenacity
   category: Physical
-  points: -4
+  points: -2
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Traits/physical.yml:48]:
```suggestion
  points: -3
```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Traits/physical.yml (hunk 2 above)
Comment (ShatteredSwords) [reply Resources/Prototypes/Traits/physical.yml:43]: So we keep Will to Live the same, but up the threshold of Will to Die and increase the points it gives you?

Comment (ShatteredSwords) [reply Resources/Prototypes/Traits/physical.yml:43]: I guess it's not a bad change, it's just weird from my perspective.

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**